model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -336,8 +343,9 @@ def _create_trial_objectives(self, trial: optuna.Trial) -> TrialObjectives:
             parameter = self._search_parameters.get_parameter(parameter_name)
 
             if parameter:
+                objective_name = f"{self._models[0].model_name()}::{parameter_name}"
```

Comment:
This is not good. We can't have this hardcoded knowledge embedded randomly here.

Reply:
I assume you are objecting to the `[0]`? This will go away when multi-model support is added. I missed adding the `TODO` above this line. Fixed now.

Reply:
No, I'm referring to the knowledge of the format model_name::parameter name. Is that just a debug string? Or is it used to identify something?

Reply:
It's just a name used when creating objectives in the Optuna library (see `_create_trial_objectives()`). I believe it's just for debug/print to the screen.

Reply:
I've refactored this to now use a method to create the trial objective name.